### PR TITLE
Use FQCN for ansible_connection in eos_smoke caching test

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: tests/integration/targets/eos_smoke/tasks/cli.yaml [hunk 1]
```diff
@@ -17,6 +17,6 @@
   tags: network_cli
 
 - name: run test cases with single_user_mode (connection=network_cli)
-  include: "{{ role_path }}/tests/cli/caching.yaml ansible_connection=network_cli ansible_network_single_user_mode=True ansible_connection=network_cli connection={{ cli }}"
+  include: "{{ role_path }}/tests/cli/caching.yaml ansible_connection=ansible.netcommon.network_cli ansible_network_single_user_mode=True ansible_connection=ansible.netcommon.network_cli connection={{ cli }}"
```
Comment (NilashishC) [inline tests/integration/targets/eos_smoke/tasks/cli.yaml:20]: The second ansible_connection is redundant.

**Comment by GomathiselviS** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**